graphqlapi/src/main/java/io/stargate/graphql/web/resources/GraphqlDdlResource.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.graphql.web.resources;
+
+import graphql.GraphQL;
+import io.stargate.graphql.web.GraphqlCache;
+import io.stargate.graphql.web.models.GraphqlJsonBody;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.container.AsyncResponse;
+import javax.ws.rs.container.Suspended;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.MediaType;
+
+@Path("/graphql-schema")
+@Singleton
+public class GraphqlDdlResource extends GraphqlResourceBase {
+
+  private final GraphQL graphql;
+
+  @Inject
+  public GraphqlDdlResource(GraphqlCache graphqlCache) {
+    this.graphql = graphqlCache.getDdl();
+  }
+
+  @GET
```

Comment:
maybe add: `@Produces(MediaType.APPLICATION_JSON)` or `APPLICATION_GRAPHQL` to have consistent headers with other HTTP methods?

Reply:
I assume you meant `@Consumes`, there's already a `@Produces(MediaType.APPLICATION_JSON)` inherited from the parent class.

I'm not sure if `@Consumes` is relevant for a GET request, since it doesn't have a body.